src/styles/blocks/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  background-size: cover;
+  background-position: center;
+  background-image: url("/image/image31.png");
+
+  @include onTablet {
+    background-image: url("/image/image3.png");
+  }
+
+  @include onDesktop {
+    background-image: url("/image/header.png");
+  }
+
+  &__content {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__title {
+    font-weight: 700;
```

Comment:
Consider using a variable for the font-weight, so it's easier to make changes in the future.